mrp_bom_attribute_match/models/mrp_bom.py

```diff
@@ -124,11 +124,15 @@ def _check_component_attributes(self):
                     )
                 )
             if not all(attr in prod_attrs for attr in comp_attrs):
+                missing_attrs = rec.env["product.attribute"].browse(
+                    [attr.id for attr in set(prod_attrs) - set(comp_attrs)]
+                )
```

Comment:
nitpicking: simpler (and opposite)

```suggestion
            missing_attrs = comp_attrs - prod_attrs
            if missing_attrs:
```